src/Data/Impact.elm

```diff
@@ -418,10 +421,44 @@ getAggregatedScoreOutOf100 { trigram } impactsPerKg =
                 -- See the documentation at https://fabrique-numerique.gitbook.io/ecobalyse/alimentaire/impacts-consideres/score-100
                 (ln 2077 - ln value) / ln 2 * 20
            )
-        |> round
+        |> floor
         |> clamp 0 100
 
 
+foodCategories : Dict String { name : String, bounds : ( Int, Int ) }
+foodCategories =
+    Dict.fromList
+        [ ( "meats", { name = "Viandes", bounds = ( 500, 4000 ) } )
+        , ( "fruitsAndVegetables", { name = "Fruits et légumes", bounds = ( 30, 450 ) } )
+        , ( "cakes", { name = "Gâteaux", bounds = ( 100, 700 ) } )
+        ]
+
+
+getAggregatedCategoryScoreOutOf100 : Definition -> String -> Impacts -> Int
+getAggregatedCategoryScoreOutOf100 { trigram } foodCategory impactsPerKg =
+    case Dict.get foodCategory foodCategories of
+        Just { bounds } ->
+            let
+                ln =
+                    logBase e
+
+                ( min, max ) =
+                    bounds
```

Comment:
Should we use the same names as in the docs, so `( impact100, impact0) = bounds` in this case? I had a hard time wrapping my head around the formula in the docs, and it didn't seem to fit with the formula I read below (took me some thinking to understand that `min` was in fact `impact100`)

Reply:
https://github.com/MTES-MCT/ecobalyse/pull/210/commits/b624b0c543a31e6bf8b9870387fb2b22d013ae90